### PR TITLE
Facade table improvements

### PR DESCRIPTION
**Pull request by g105b**
Hi @speich ,

Please see these minor changes I've made to the HTMLTableElement tests. I'd like to hear your feedback.

Cheers,
Greg.

### CODE REVIEW

**Comment by speich** [comment]
Sorry for the silence. I'll have a look this weekend.

**Comment by g105b** [comment]
No rush! Take you time, I've still a few classes to complete coding before the facade will be released.

File: src/HTMLElement/HTMLTableElement.php [hunk 1]
```diff
@@ -307,12 +307,15 @@ protected function __prop_get_rows():HTMLCollection {
 			$col = $this->getElementsByTagName('tr');
 			foreach($col as $row) {
 				$name = strtolower($row->parentNode->nodeName);
```
Comment (speich) [inline src/HTMLElement/HTMLTableElement.php:309]: I would move this line now to line 318 just before the switch statement

**Review by speich — CHANGES_REQUESTED** [review]
I would make a tiny change, see inline comment

**Comment by g105b** [comment]
Thanks @speich - nearly there now!

**Pull request merged**